### ISSUE TITLE
关于self.get_reliable函数的报错问题

### ISSUE THREAD

**Issue by codinglin** [Huang-ShiRui/Semi-UIR#7]
![aab81692e6370201cb281b9a4bf089d](https://user-images.githubusercontent.com/42762318/232415387-d404c369-18cb-475b-a5db-9f693d290785.png)
score_t = self.iqa_metric(teacher_predict).detach().cpu().numpy()
score_s = self.iqa_metric(student_predict).detach().cpu().numpy()
self.iqa_metric只会生成一个值，如何进行如下遍历操作
for idx in range(0, N):
      if score_t[idx] > score_s[idx]:
            if score_t[idx] > score_r[idx]:

**Comment by codinglin** [comment]
![e4cf641c6400753d3eafdefc8937a02](https://user-images.githubusercontent.com/42762318/232416238-1461a231-9497-4035-ab21-e9b05dd15114.png)
希望能得到您的回复，我们对您的工作很感兴趣

**Comment by Star-xing1** [comment]
降低一下pyiqa的版本

> ![e4cf641c6400753d3eafdefc8937a02](https://user-images.githubusercontent.com/42762318/232416238-1461a231-9497-4035-ab21-e9b05dd15114.png) 希望能得到您的回复，我们对您的工作很感兴趣

**Comment by codinglin** [comment]
> 降低一下pyiqa的版本
> 
> > ![e4cf641c6400753d3eafdefc8937a02](https://user-images.githubusercontent.com/42762318/232416238-1461a231-9497-4035-ab21-e9b05dd15114.png) 希望能得到您的回复，我们对您的工作很感兴趣

请问下您的版本号是多少？

**Comment by Star-xing1** [comment]
> > 降低一下pyiqa的版本
> > > ![e4cf641c6400753d3eafdefc8937a02](https://user-images.githubusercontent.com/42762318/232416238-1461a231-9497-4035-ab21-e9b05dd15114.png) 希望能得到您的回复，我们对您的工作很感兴趣
> 
> 请问下您的版本号是多少？

0.1.4

**Comment by codinglin** [comment]
> > > 降低一下pyiqa的版本
> > > > ![e4cf641c6400753d3eafdefc8937a02](https://user-images.githubusercontent.com/42762318/232416238-1461a231-9497-4035-ab21-e9b05dd15114.png) 希望能得到您的回复，我们对您的工作很感兴趣
> > 
> > 
> > 请问下您的版本号是多少？
> 
> 0.1.4

非常感谢，已经解决了

**Comment by codinglin** [comment]
请问您的版本是多少，我按照您的版本来




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Huang-ShiRui/Semi-UIR"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年4月17日(星期一) 下午4:20
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Huang-ShiRui/Semi-UIR] 关于self.get_reliable函数的报错问题 (Issue #7)





 
降低一下pyiqa的版本
  
 希望能得到您的回复，我们对您的工作很感兴趣

**Comment by kkllww** [comment]
你好，我想问一下降低版本后可以运行，但是为什么更换训练集和测试集后也会出现这样的报错。
